content/commands/ts.add/index.md

```diff
@@ -177,6 +178,30 @@ This override is effective only for this single command and does not set the tim
 This argument has no effect when a new time series is created by this command.
 </details>
 
+<details open><summary><code>IGNORE ignoreMaxTimediff ignoreMaxValDiff</code></summary> 
+
+is the policy for handling insertion of samples that may be duplicates of existing samples according to the following rules:
```

Comment:
"duplicate of existing samples" may be misleading. We only compare the new sample with the most recent (highest timestamp) sample (see logical condition).
(This means that duplicates would be filtered only if samples are reported in-order).